optuna/importance/_fanova/_fanova.py

```diff
@@ -12,14 +12,10 @@
 This stands in contrast to the original implementation which is partially written in C++.
 Since Python runtime overhead may become noticeable, included are instead several
 optimizations, e.g. vectorized NumPy functions to compute the marginals, instead of keeping all
-running statistics. Known cases include assessing higher order importances, e.g. pairwise
-importances, this is due to the fact that the number of partitions to visit grows exponentially,
-or when assessing categorical features with a larger number of choices since each choice is
-given a unique one-hot encoded raw feature.
-
+running statistics. Known cases include when assessing categorical features with a larger
```

Comment:
Nit.
```suggestion
running statistics. Known cases include assessing categorical features with a larger
```

Reply:
Thank you :bow: I applied it at e9a0955.